operation.go

```diff
@@ -437,7 +437,7 @@ func parseMimeTypeList(mimeTypeList string, typeList *[]string, format string) e
 	return nil
 }
 
-var routerPattern = regexp.MustCompile(`([\w\.\/\-{}\+]+)[^\[]+\[([^\]]+)`)
+var routerPattern = regexp.MustCompile(`([\w\.\/\-{}\+:]+)[^\[]+\[([^\]]+)`)
```

Comment:
Please fix the regexp accordigly 
The following routes should not be valid 
"/@Router :customer/get-wishlist/{wishlist_id}:move [post]"
[path format](https://tools.ietf.org/html/std66#section-3.3) 
```
In addition, a URI reference (Section 4.1) may be a relative-path reference, in which case the first path segment cannot contain a colon (":") character.
```